### PR TITLE
RATEST-151: E2E Vitals and triaging workflow

### PR DESCRIPTION
**Pull request by kdaud**
Ticket [ID](https://issues.openmrs.org/browse/RATEST-151)
Description -> E2E Vitals and triaging workflow

### CODE REVIEW

File: src/test/java/org/openmrs/contrib/qaframework/automation/VitalsAndTriagingSteps.java [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+	  patientCaptureVitalsPage.setTemperatureField("35.8");
+	  patientCaptureVitalsPage.setPulseField("124");
+	  patientCaptureVitalsPage.setRespiratoryField("115");
+	  patientCaptureVitalsPage.setBloodPressureFields("122", "75");
+	  patientCaptureVitalsPage.setBloodOxygenSaturationField("54");
+	}
+
+	@And("a user clicks on save button")
+	public void savePatientVitals() {
+	  patientCaptureVitalsPage.confirm();
```
Comment (k-joseph) [inline src/test/java/org/openmrs/contrib/qaframework/automation/VitalsAndTriagingSteps.java:61]: please confirm the existing indention and correspond to it throughout this file

File: src/test/java/org/openmrs/contrib/qaframework/automation/VitalsAndTriagingSteps.java [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+	  patientCaptureVitalsPage.save();
+	}
+
+	@Then("the system adds patient vitals into the vitals table")
+	public void systemAddsPatientVitals() {	  
+	  assertTrue(patientCaptureVitalsPage.containsText("Vitals"));
```
Comment (k-joseph) [inline src/test/java/org/openmrs/contrib/qaframework/automation/VitalsAndTriagingSteps.java:67]: i guess this is no where else on the page until Vitals exist!

File: src/test/resources/features/refapp-2.x/03-trackPatient/vitalsAndTriaging.feature [hunk 3]
```diff
@@ -0,0 +1,12 @@
+Feature: Vitals Management
+
+  Background:
+    Given a user clicks on Capture Vitals link from Patient dashboard
+    Then the system loads Vitals page
+
+  @selenium
+  @vitals
+  Scenario: Capture Patient vitals
```
Comment (k-joseph) [inline src/test/resources/features/refapp-2.x/03-trackPatient/vitalsAndTriaging.feature:9]: Please add extra scenarios to handle success/failure and validation cases

Comment (kdaud) [reply src/test/resources/features/refapp-2.x/03-trackPatient/vitalsAndTriaging.feature:9]: > add extra scenarios to handle success/failure and validation cases

a little more light on this and then I act accordingly

File: src/test/java/org/openmrs/contrib/qaframework/automation/VitalsAndTriagingSteps.java (hunk 2 above)
Comment (kdaud) [reply src/test/java/org/openmrs/contrib/qaframework/automation/VitalsAndTriagingSteps.java:67]: > i guess this is no where else on the page until Vitals exist!

After carrying out investigation on the page, I stand to agree with your guess.

This suggests to me that idea behind the commented `Code Block` might be left out. However, from the technical-end, I think the first function should be implemented to  properly mimic the end user experience on the feature. 
{

`patientCaptureVitalsPage.showPatientVitals();`
then
`assertTrue(patientCaptureVitalsPage.containsText("Vitals"));	`

}

File: src/test/resources/features/refapp-2.x/03-trackPatient/vitalsAndTriaging.feature (hunk 3 above)
Comment (kaweesi) [reply src/test/resources/features/refapp-2.x/03-trackPatient/vitalsAndTriaging.feature:9]: Christine will be editing the description to provide more context on this, you can reach-out to her for more feedback

**Comment by kdaud** [comment]
I messed up with the PR and decided to close it

**Pull request closed without merging**